mock-core-configs/etc/mock/templates/centos-7.tpl

```diff
@@ -1,11 +1,15 @@
 # This list is taken from 'epel-7-x86_64' @buildsys-build group, minus the
 # 'epel-*' specific stuff.
 config_opts['chroot_setup_cmd'] = 'install bash bzip2 coreutils cpio diffutils findutils gawk gcc gcc-c++ grep gzip info make patch redhat-rpm-config rpm-build sed shadow-utils tar unzip util-linux which xz'
+config_opts['chroot_additional_packages'] = 'scl-utils-build'
```

Comment:
This package is not installed by default in Koji EPEL, so we can not add it here.

Reply:
Does Koji EPEL have the SCLo repositories (and enabled by default), as the Mock configs do?
The repo definitions in Mock alone aren't enough to build packages that use SCLo. I would suggest we consider dropping them if we aren't going to have the corresponding RPM macros. Or this could be moved to a separate config.

Reply:
Yes.

Reply:
But scl-utils-build is not in the buildroot, note this: https://github.com/rpm-software-management/mock/blob/691d914f561ca164ad777bd456aa9ea8011d0650/mock-core-configs/etc/mock/templates/centos-7.tpl#L87